src/point.h

```diff
@@ -65,6 +65,12 @@ struct point {
         return point( std::abs( x ), std::abs( y ) );
     }
 
+    // Dummy implementation of raw() to allow reasoning about
+    // abstract generic points.
+    constexpr point raw() const {
+        return point( x, y );
```

Comment:
```suggestion
        return point( x, y );
```
Shouldn't these just be `return this`?

Reply:
Reading a bit more, I see they are indeed equivalent.